### PR TITLE
improving up command when service already exists

### PR DESCRIPTION
**Pull request by zyndiecate**
fixes https://github.com/giantswarm/cli/issues/372

deps
- https://github.com/giantswarm/client-go/pull/94

Now we don't start the service when it is already up.
```
$ swarm --api-endpoint=http://192.168.33.101:8080/v1/ up world/swarm.json
Service 'world' already exists in environment 'giantswarm/dev'.
Service 'world' is already up.
```

When the service is down we start it as usual but additionally show some more information as requested in the original issue.
```
$ swarm --api-endpoint=http://192.168.33.101:8080/v1/ up world/swarm.json
Service 'world' already exists in environment 'giantswarm/dev'.
Starting service 'world'…
Waiting for 'world' to get started…
Service 'world' is up.
You can see all components using this command:

    swarm status world

```

### CODE REVIEW

**Comment by zyndiecate** [comment]
RFR @ewoutp

File: up.go [hunk 1]
```diff
@@ -84,14 +85,25 @@ func processUp(configFile string) {
 	}
 
 	// See if the service exists
-	_, err = client.AppStatus(serviceName)
-	if clientgo.IsNotFound(err) {
+	status, err := client.AppStatus(serviceName)
```
Comment (ewoutp) [inline up.go]: We should rename AppStatus to ServiceStatus #notinthispr

**Comment by ewoutp** [comment]
LGTM

**Comment by marians** [comment]
LGTM2

File: up.go (hunk 1 above)
Comment (zyndiecate) [inline up.go]: +1

**Pull request merged**